providers/src/airflow/providers/edge/cli/edge_command.py

```diff
@@ -238,14 +238,18 @@ def check_running_jobs(self) -> None:
                     EdgeJob.set_state(job.edge_job.key, TaskInstanceState.FAILED)
             if job.logfile.exists() and job.logfile.stat().st_size > job.logsize:
                 with job.logfile.open("r") as logfile:
+                    push_log_chunk_size = conf.getint("edge", "push_log_chunk_size", fallback=1024)
```

Comment:
Fallback is not needed if the config entry is in provider YAML defined.
```suggestion
                    push_log_chunk_size = conf.getint("edge", "push_log_chunk_size")
```

Reply:
done